engine/table/src/main/java/io/deephaven/engine/table/impl/by/ChunkedOperatorAggregationHelper.java

```diff
@@ -626,7 +626,7 @@ private void propagateRemovesToOperators(@NotNull final RowSequence keyIndicesTo
             boolean firstOperator = true;
             setFalse(modifiedSlots, runStarts.size());
 
-            final Chunk<? extends Values>[] valueChunks = findRuns ? workingChunks : new Chunk[ac.size()];
+            final Chunk<? extends Values>[] valueChunks = new Chunk[ac.size()];
```

Comment:
Why change this? I don't see anything wrong with the previous, lower-allocation version. I do see that in other parts of the code you couldn't do this due to where the `findSlotRuns` call happens... are you just trying to be consistent everywhere?
Maybe would be better to just allocate this on the context?

Reply:
It was for consistency, but I moved it to the context.